state/store.go

```diff
@@ -706,3 +748,14 @@ func responseFinalizeBlockFromLegacy(legacyResp *cmtstate.LegacyABCIResponses) *
 		// be caught and filled in consensus/replay.go
 	}
 }
+
+func int64FromBytes(bz []byte) int64 {
```

Comment:
I wonder whether a test unit could be added to verify whether saved and loaded `int64` values are the same.

Reply:
Done!